homeassistant/components/knx/config_flow.py

```diff
@@ -1,29 +1,41 @@
 """Config flow for KNX."""
 from __future__ import annotations
 
-from typing import Final
+from typing import Any, Final
 
 import voluptuous as vol
 from xknx import XKNX
 from xknx.io import DEFAULT_MCAST_GRP, DEFAULT_MCAST_PORT
 from xknx.io.gateway_scanner import GatewayDescriptor, GatewayScanner
 
 from homeassistant import config_entries
+from homeassistant.config_entries import ConfigEntry, OptionsFlow
 from homeassistant.const import CONF_HOST, CONF_PORT
+from homeassistant.core import callback
 from homeassistant.data_entry_flow import FlowResult
 import homeassistant.helpers.config_validation as cv
 
 from .const import (
     CONF_KNX_AUTOMATIC,
     CONF_KNX_CONNECTION_TYPE,
     CONF_KNX_INDIVIDUAL_ADDRESS,
+    CONF_KNX_INITIAL_CONNECTION_TYPES,
     CONF_KNX_ROUTING,
     CONF_KNX_TUNNELING,
     DOMAIN,
 )
 from .schema import ConnectionSchema
 
 CONF_KNX_GATEWAY: Final = "gateway"
+CONF_MAX_RATE_LIMIT: Final = 60
```

Comment:
should we move that to const.py so everything is in one place?

Reply:
We don't need that anywhere else so I don't believe we need this in const.py